package.json

```diff
@@ -10,6 +10,11 @@
     "node": ">=16.4.0"
   },
   "dependencies": {
+    "@babel/plugin-transform-react-display-name": "7.18.6",
```

Comment:
What are these needed for now? It doesn't look like we're actually using them since I'd assume we'd have to update the babel config for that